ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkReader.java

```diff
@@ -279,9 +296,13 @@ private Void readShunt(String[] tokens) {
         }
 
         if (sc.getModelType() == ShuntCompensatorModelType.NON_LINEAR) {
-            throw new PowsyblException("non linear shunt not supported yet");
+            // nothing.
+            if (sections == 0) {
+                sc.setSectionCount(sections);
+            } // else nothing.
+        } else {
```

Comment:
Maybe log that we only disconnect the shunt but don't do anything else

Reply:
Yes okay. Indeed at that stage of the PR, I have converted a non linear shunt as a one section shunt. The AMPL tool has the options: use the shunt as it is or disconnect it. I think that we have to improve the conversion but I am not able to do it by myself.